trio/subprocess.py

```diff
@@ -0,0 +1,6 @@
+from ._subprocess import Process, run
+from subprocess import (
+    PIPE, STDOUT, DEVNULL, CalledProcessError, SubprocessError, TimeoutExpired,
+    CompletedProcess
+)
```

Comment:
This is leaving out a bunch of Windows-specific things like `STARTUPINFO`, `CREATE_NEW_CONSOLE`, etc. [List is here](https://docs.python.org/3/library/subprocess.html#windows-popen-helpers).